pkg/storage/formattedstore/batchwrite.go

```diff
@@ -148,10 +164,13 @@ func newPendingBatch(provider batchProvider, formatter Formatter) *pendingBatch
 	return &pendingBatch{
 		provider:  provider,
 		formatter: formatter,
+		size:      0,
```

Comment:
Does 0 mean no maximum or disable batch?

Should also have a way to indicate no maximum.

Reply:
I will add case for no maximum

Reply:
done